app/controllers/listings_controller.rb

```diff
@@ -0,0 +1,9 @@
+class ListingsController < ApplicationController
```

Comment:
Nice! Im glad were starting the transition. All the pending tests are a pain tho.